src/main/java/frc/robot/Constants.java

```diff
@@ -42,110 +52,35 @@ public static class constDrivetrain {
     // TODO: Convert all applicable fields to MEASUREs
 
     // In Rotations: Obtain by aligning all of the wheels in the correct direction
-    // and
-    // copy-pasting the Raw Absolute Encoder value
+    // and copy-pasting the Raw Absolute Encoder value
     public static final double FRONT_LEFT_ABS_ENCODER_OFFSET = 0.417236;
     public static final double FRONT_RIGHT_ABS_ENCODER_OFFSET = -0.254395;
     public static final double BACK_LEFT_ABS_ENCODER_OFFSET = 0.258789;
     public static final double BACK_RIGHT_ABS_ENCODER_OFFSET = -0.290039;
 
-    public static final double WHEEL_DIAMETER = 0.09779;
-    public static final Distance WHEEL_RADIUS = Units.Meters.of(WHEEL_DIAMETER / 2);
-    public static final double WHEEL_CIRCUMFERENCE = WHEEL_DIAMETER * Math.PI;
-
-    // Taken from the online listing
-    public static final double DRIVE_GEAR_RATIO = 6.75;
-    public static final double STEER_GEAR_RATIO = 150.0 / 7.0;
+    public static final SN_SwerveConstants SWERVE_CONSTANTS = new SN_SwerveConstants(
+        SN_SwerveConstants.MK4I.FALCON.L3.steerGearRatio,
+        0.09779 * Math.PI,
+        SN_SwerveConstants.MK4I.FALCON.L3.driveGearRatio,
+        SN_SwerveConstants.MK4I.FALCON.L3.maxSpeedMeters);
 
-    /**
-     * <p>
-     * Theoretical maximum translational speed while manually driving on the
-     * Competition Robot.
-     * </p>
-     * <b>Units:</b> Meters Per Second
-     */
-    public static final double THEORETICAL_MAX_DRIVE_SPEED = SN_SwerveConstants.MK4I.FALCON.L3.maxSpeedMeters;
+    public static final double WHEEL_DIAMETER = SWERVE_CONSTANTS.wheelCircumference / Math.PI;
+    public static final Distance WHEEL_RADIUS = Units.Meters.of(WHEEL_DIAMETER / 2);
```

Comment:
In the future we should probably make these getter funtions inside of the SuperCORE SwerveConstants object